### PR TITLE
Update billing to reflect new changes

### PR DESCRIPTION
**Pull request by ivanaur**

### CODE REVIEW

File: billing_5c06ad9604286304a71cfa5b.md [hunk 1]
```diff
@@ -1,26 +1,41 @@
 With Semaphore 2.0 you will only pay for what you actually used and you can
 customize your usage according to your team's needs.
 
+## Trial Period
+
+Once you sing up, your first organization will be automatically created based on your
```
Comment (milica-nerlovic) [inline billing_5c06ad9604286304a71cfa5b.md]: sign

File: billing_5c06ad9604286304a71cfa5b.md [hunk 2]
```diff
@@ -1,26 +1,41 @@
...
+GitHub username and you will be in a 14-day trial period. During this period you will be 
+able to use [the default quota of machines](https://docs.semaphoreci.com/article/133-quotas-and-limits) completely free of charge.
```
Comment (milica-nerlovic) [inline billing_5c06ad9604286304a71cfa5b.md]: `default machine quota` or `default quota for machines`

File: billing_5c06ad9604286304a71cfa5b.md [hunk 3]
```diff
@@ -1,26 +1,41 @@
...
+Once you sing up, your first organization will be automatically created based on your
+GitHub username and you will be in a 14-day trial period. During this period you will be 
+able to use [the default quota of machines](https://docs.semaphoreci.com/article/133-quotas-and-limits) completely free of charge. 
+
+Of course, you can easily scale up if you need a bigger capacity by going to **Billing > 
+See detailed insights… > Quota > Request upgrade…** or by reaching out to 
+[customersuccess@semaphoreci.com](mailto:customersuccess@semaphoreci.com) (please 
+include which type of machine you prefer).
+
+If you don't want to use trial period for the automatically created organization,
```
Comment (milica-nerlovic) [inline billing_5c06ad9604286304a71cfa5b.md]: the trial period

File: billing_5c06ad9604286304a71cfa5b.md [hunk 4]
```diff
@@ -1,26 +1,41 @@
...
+
 ## Plans
 
-Once you sign up and create an organization, you are automatically on a
-**free plan**. With this plan, you will get $20 free credit every 30 days for
-the first organization you create on Semaphore 2.0. If needed, an owner of
-the organization can request a transfer of $20 free credit to another organization
-he owns by reaching out to [support@semaphoreci.com](mailto:support@semaphoreci.com).
+After your trial period expires, your organization will be blocked and you will be 
+prompted to choose **Free plan** or **Paid plan**.
```
Comment (milica-nerlovic) [inline billing_5c06ad9604286304a71cfa5b.md]: a free or a paid plan

File: billing_5c06ad9604286304a71cfa5b.md [hunk 5]
```diff
@@ -1,26 +1,41 @@
...
+
+**Free plan** includes $10 free credit per calendar month and you will receive it on 
+the 1st of every month. On this plan, you will have parallelism 1 and you can use
```
Comment (milica-nerlovic) [inline billing_5c06ad9604286304a71cfa5b.md]: `parallelism of 1` or `you will be able to run 1 job in parallel`

**Comment by milica-nerlovic** [comment]
Left some grammar comments, other than that, great! :rocket: Thanks for taking care of it.

File: billing_5c06ad9604286304a71cfa5b.md [hunk 6]
```diff
@@ -1,26 +1,41 @@
...
+You can have an unlimited number of organizations, but only one of the organizations
+you own can be on a free plan.
 
-If you decide to add your credit card, you will be switched to the
-**paid plan**. Once you do that, you won’t be charged anything. The first
-charge will happen at the end of the billing period which are 30 days long
-starting from the moment you created an organization. When you add your credit
-card, you will receive $200 gift credit for your organization that you can use
-during a 3-month period.
+**Paid plan** is activated once you add your credit card and you will be charged
```
Comment (shiroyasha) [inline billing_5c06ad9604286304a71cfa5b.md]: **Paid plan** is activated once you add your credit card. You will ....

File: billing_5c06ad9604286304a71cfa5b.md (hunk 5 above)
Comment (shiroyasha) [reply billing_5c06ad9604286304a71cfa5b.md]: It is not clear "what will I receive on the 1st every month".

Alternative:

> **Free plan** includes $10 free credit per calendar month. At the beginning of each month, you will receive a new $10 credit for your organization.

**Pull request merged**